engine/env/spark/src/test/resources/conf_hdfs.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<configuration xmlns="http://eobjects.org/analyzerbeans/configuration/1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<configuration xmlns="http://eobjects.org/analyzerbeans/configuration/1.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 
 	<datastore-catalog>
-		<csv-datastore name="person_names">
-			<filename>hdfs://bigdatavm:9000/datacleaner/test/person_names.txt</filename>
-			<multiline-values>false</multiline-values>
-		</csv-datastore>
+		<json-datastore name="person_data">
```

Comment:
But we're not restricted to having only one datastore?! Just create another one and make a job that refers to that?